README.md

```diff
@@ -0,0 +1,219 @@
+- __Management is important and valuable__: Great managers can make a positive difference in people's lives and can materially impact the success of a team or a business. Bad management and lack of management can be equally destructive. Try to provide a great experience to everyone you work with. An important distinction here is to not focus on trying to make everyone like you all the time. Great management often means telling people hard truths. Which leads into...
+- __Feedback is vital:__ Give pointed, direct feedback on a regular basis. Don’t wait for an annual review to help someone improve. Try to generalize negative feedback and be specific with positive feedback. Additionally, seek out feedback from those you work with, in all directions, in order to drive your own growth.
+- __Diverse teams make the best decisions__: This may seem obvious but software industry demographics indicate it is not. Seek out a variety of backgrounds and talent profiles when building a team. Strive for great communication to unlock the power of these different experiences.
+
+## Expectations
+
+### What you can expect of me
+
+Here is a short list of behaviors you should expect from me at all times. If I'm ever failing to meet any of the expectations outlined in this section, please let me know immediately. This is not an exhaustive list of expectations but aims to cover those that are most vital.
+
+#### You will be a top priority
+
+It's my belief that the people who report to me are _the job_. I'm tasked with various other responsibilities as an Engineering Manager but none are higher priority in my view than ensuring my direct reports are getting what they need to be successful. In practice, this means you should expect that I will make time for you when you need it, follow through on commitments I make to you, and help you deal with whatever challenges you encounter.
+
+#### Give my best in everything I do
+
+This is a foundational principle for how I approach everything in my life. If I'm going to do something, I try to do it as well as I can. This doesn't mean that I will always deliver outstanding results, just that I'll aim to do my best. In practice, this means you should expect me to give you a great effort on every front from our 1:1 meetings to performance evaluations to helping you navigate challenges.
+
+#### Default to transparent
+
+I believe organizations operate at their highest level when information is not restricted. In the interest of this, I commit to being transparent and to proactively share as much as I can. The flip side of this is that you should feel epowered to ask me for information. Two important exceptions to this policy of transparency are sensitive personal information and conversations around performance; I will always keep these confidential.
```

Comment:
I always felt like you were being transparent and that made the relationship feel very much like a relationship :) what do you consider to be a part of "conversations around performance"? Is that regarding just other people's performance or mine as well? How do I get feedback and improve if the latter?

Typo in  "epowered"

Reply:
Both directions! I won't discuss performance conversations that we have with other people. I also won't discuss with you the performance conversations I have with other people.